frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -178,14 +176,7 @@ public void connect() throws WorkerInitializationException {
                                 @Override
                                 public void initChannel(Channel ch) {
                                     ChannelPipeline p = ch.pipeline();
-                                    p.addLast(
-                                            new DelimiterBasedFrameDecoder(
-                                                    81920000,
-                                                    Delimiters
-                                                            .lineDelimiter())); // TODO: Make this config based
-                                    p.addLast(STRING_DECODER);
-                                    p.addLast(MSG_DECODER);
-                                    p.addLast(MSG_ENCODER);
+                                    p.addLast(msgCodec);
```

Comment:
new MessageCodec()

Reply:
Any reason not to have it at the top?

Reply:
The codec can't be shared. It should be created per pipeline(channel)

Reply:
Makes sense. If each worker can serve multiple models, then this would cause weird problems. Will make the change.